tracpro/baseline/migrations/0005_auto_20150817_1547.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think that these 5 migrations can be squashed into one initial migration. 

(I would do it by deleting all `baseline` migrations and re-running `django-admin makemigrations baseline` to create a brand new initial migration. However [squashmigrations](https://docs.djangoproject.com/en/1.8/topics/migrations/#squashing-migrations) looks like it does essentially the same thing.)